swagger-editor-dist-package/README.md

```diff
@@ -1,3 +1,22 @@
+# swagger-editor-dist
+
 This module, `swagger-editor-dist`, exposes Swagger-Editor's entire dist folder as a dependency-free npm module.
```

Comment:
Update the "depdendency free" claim.
->
`...exposes Swagger-Editor's entire dist folder as an almost (see [anonymized analytics](#anonymized-analytics)) dependency-free npm module.`